setup.py

```diff
@@ -17,7 +17,7 @@
 install_requires = [
     'django-modeldict>=1.2.0',
     'nexus>=0.2.3',
-    'django-jsonfield>=0.8.0,<0.9.4',
+    'jsonfield==0.9.23',
```

Comment:
I thought `django-jsonfield` didn't have Django 1.7 compatibility, but it's just the `0.8.x` series that doesn't have compatiblity. I just force pushed and reverted back to `django-jsonfield`, but bumped the version numbers